modules/spring-data/src/main/java/org/apache/ignite/springdata/repository/support/ConditionFalse.java

```diff
@@ -0,0 +1,18 @@
+package org.apache.ignite.springdata.repository.support;
+
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+
+/**
+ * Always false condition.
+ * Tells spring context never load bean with such Condition.
+ *
+ * @author Roman_Meerson
+ */
+public class ConditionFalse implements Condition {
+    @Override
```

Comment:
`@Override` annotation should be placed on the same line as method signature.
See https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-@Annotations